### PR TITLE
If we're checking for jQuery, use jQuery

### PR DESCRIPTION
**Pull request by dotsam**
Use fully qualified jQuery to register function, don't assume that noConflicts mode isn't enabled

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   ],
   "description": "Library to pass Sass variables via CSS to JavaScript",
   "main": [
-    "js/dist/sass-to-js.min.js",
+    "js/src/sass-to-js.js",
```
Comment (malyw) [inline bower.json]: Why this change is needed?
Usually the dist/ version is used, e.g. for jQuery: https://github.com/jquery/jquery/blob/master/package.json

**Comment by dotsam** [comment]
The bower spec says that `main` shouldn't contain minified versions, and warnings to that effect are thrown on install. https://github.com/bower/spec/blob/master/json.md#main

**Comment by malyw** [comment]
I propose to either leave it as it is to be consistent with package.json (many packages use dist/ like https://github.com/angular-translate/angular-translate/blob/master/bower.json)
or just consider removing bower.json.
bower.json change is also out of the scope of the PR name, so can be a good idea to revert this change, so PR can be easily merged.

**Comment by dotsam** [comment]
I've rolled back this branch to only contain the jquery change. As the bower issue only generates a warning, I'm fine leaving it as-is. Thanks.

**Comment by malyw** [comment]
Thanks, Sam, merged
